utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/hostapd_ubus_stations.lua

```diff
@@ -55,9 +56,10 @@ local function scrape()
     metric_hostapd_ubus_station_rrm_caps_ftm_range_report(label_station, rrm_caps_ftm_range_report)
   end
 
-  for _, ifname in ipairs(get_wifi_interfaces()) do
+  for _, hostapd_int in ipairs(get_wifi_interfaces()) do
     local u = ubus.connect()
-    local clients_call = u:call("hostapd." .. ifname, "get_clients", {})
+    local clients_call = u:call(hostapd_int, "get_clients", {})
+    local ifname = hostapd_int:gsub("hostapd.", "")
```

Comment:
If I read `get_wifi_interfaces()` correctly, you are returning `ifname`, not `hostapd.ifname`, so no need to change anything

Reply:
I changed that now, that I return hostapd names.